### DISCUSSION TITLE
Microsoft.CodeAnalysis.CSharp  - 4.9.2

### DISCUSSION THREAD

**Discussion by rmanjusha** [elsa-workflows/elsa-core#5374] [Q&A]
Any plans to upgrade Microsoft.CodeAnalysis.CSharp to latest ? I am facing version conflicts since Elsa.CSharp used lower version (4.8.0)

**Comment by glime-ai** [comment]
To provide an accurate response regarding the plans to upgrade Microsoft.CodeAnalysis.CSharp to the latest version in the context of Elsa Workflows, I would need to check the latest discussions, issues, and commits in the official Elsa Workflows GitHub repository. However, based on the context provided, there is no direct mention of plans to upgrade Microsoft.CodeAnalysis.CSharp.

If you are looking for specific information about this upgrade, I recommend checking the Elsa Workflows GitHub repository for the most recent issues, pull requests, or discussions that might address this topic. You can also consider opening a new issue or discussion in the repository to inquire directly about the upgrade plans for Microsoft.CodeAnalysis.CSharp.

Would you like me to provide instructions on how to search for this information in the GitHub repository, or do you have any other questions related to Elsa Workflows?

This comment was generated by [Glime](https://glimelab.ai/?utm_source=dRbvTb7IhCSYiI&utm_medium=github).
